foundationdb/src/transaction.rs

```diff
@@ -754,6 +799,68 @@ impl Future for TrxWatch {
     }
 }
 
+/// A versionstamp is a 10 byte, unique, monotonically (but not sequentially) increasing value for
+/// each committed transaction. The first 8 bytes are the committed version of the database. The
+/// last 2 bytes are monotonic in the serialization order for transactions.
+#[derive(Clone, Copy)]
+pub struct Versionstamp([u8; 10]);
```

Comment:
I’m confused about this size, is it a context thing? The java client claims this is 12: https://apple.github.io/foundationdb/javadoc/index.html?com/apple/foundationdb/tuple/Versionstamp.html

Is it a context thing? Where in the tuple with the pack format it has the marker code + 10 bytes, but the raw structure is just 10?

Reply:
It seems that API versionstamp and Tuple versionstamp has different size: https://forums.foundationdb.org/t/implementing-versionstamps-in-bindings/250

Reply:
We’ll need to be careful with this one :)